optimize.c

```diff
@@ -2138,48 +2138,48 @@ convert_code_r(compiler_state_t *cstate, conv_state_t *conv_state,
 	    {
 		u_int i;
 		int jt, jf;
-		const char *ljerr = "%s for block-local relative jump: off=%d";
```

Comment:
What happens with

    static const char ljerr[] = "%s for block-local relative jump: off=%d";